pkg/kube/volume/volume_test.go

```diff
@@ -237,27 +237,6 @@ func (s *TestVolSuite) TestZoneToRegion(c *C) {
 		},
 	} {
 		reg := zonesToRegions(tc.zone)
-		c.Assert(slicesEqual(reg, tc.expectedRegion), Equals, true)
+		c.Assert(reg, DeepEquals, tc.expectedRegion)
```

Comment:
@e-sumin do we know if deep equal returns true if two slices have same elements but in diff order? because that the scenario here and that's the reason I wrote that custom slice checker.